packages/xrpl/src/sugar/getOrderbook.ts

```diff
@@ -30,44 +29,26 @@ const getOrderbookOptionsSet = new Set([
   'taker',
 ])
 
+export interface GetOrderBookOptions {
```

Comment:
I think `Options` should have docstrings for each field since this is exported and used elsewhere.

Reply:
Fixed